src/hotspot/share/opto/stringopts.cpp

```diff
@@ -218,10 +185,7 @@ class StringConcat : public ResourceObj {
   void maybe_log_transform() {
     CompileLog* log = _stringopts->C->log();
     if (log != NULL) {
-      log->head("replace_string_concat arguments='%d' string_alloc='%d' multiple='%d'",
-                num_arguments(),
-                _string_alloc != NULL,
-                _multiple);
+      log->head("replace_string_concat arguments='%d'", num_arguments());
```

Comment:
Why did you remove the additional information from the logging message?

Reply:
they are only informative. they won't change behavior. should I keep them?

Reply:
hi, @TobiHartmann , 
I bring back multiple. _string_alloc is always NULL. there's no way to set this member data.  so I delete it.